src/tests/ftest/daos_test/daos_core_test.py

```diff
@@ -76,7 +76,7 @@ def test_subtest(self):
 
         Use Cases: core tests for daos_test
 
-        :avocado: tags=daos_test,multiserver,vm,regression
+        :avocado: tags=pr,hw,unittest,medium,daos_test
```

Comment:
should we just do hw only for the core tests?

Reply:
@jolivier23: Not sure which way to read your question, but the tag change here does mean that we are only running `daos_test` on the hardware cluster.

Reply:
This patch is to add the hardware stage.  Adding / changing the tests for it would be a different patch.